quic/s2n-quic-rustls/Cargo.toml

```diff
@@ -10,6 +10,7 @@ license = "Apache-2.0"
 exclude = ["corpus.tar.gz"]
 
 [dependencies]
+bytes = "1"
```

Comment:
I don't think we need the default features for this, right?

Reply:
`on_application_protocol(Bytes)` means the tls libs need to know about and construct Bytes now.

Reply:
right; but we should be able to `bytes = { version = "1", default-features = false }`